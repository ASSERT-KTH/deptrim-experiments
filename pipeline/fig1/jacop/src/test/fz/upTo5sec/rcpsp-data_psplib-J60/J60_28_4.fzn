predicate jacop_cumulative(array [int] of var int: s,array [int] of var int: d,array [int] of var int: r,var int: b);
array [1..2] of int: X_INTRODUCED_62 = [1,-1];
array [1..46] of int: X_INTRODUCED_119 = [4,9,8,7,3,1,5,9,9,1,7,6,3,9,8,8,5,6,9,1,3,9,6,1,7,10,3,9,5,5,10,2,9,10,2,8,2,4,5,10,7,5,1,10,8,7];
array [1..46] of int: X_INTRODUCED_120 = [1,9,3,4,3,10,5,5,4,5,7,8,5,9,5,5,7,9,8,9,3,2,5,1,4,2,9,1,1,7,10,10,1,5,8,10,8,2,1,7,9,4,1,8,9,5];
array [1..46] of int: X_INTRODUCED_122 = [9,2,2,8,10,1,9,2,7,10,6,2,3,2,5,7,4,2,9,10,1,1,10,10,7,7,7,10,7,9,4,8,4,4,6,3,1,2,9,7,1,1,2,1,8,5];
array [1..46] of int: X_INTRODUCED_123 = [9,3,4,3,5,5,4,5,3,7,8,7,5,9,5,7,9,9,4,8,9,1,3,2,6,5,10,1,1,1,10,4,1,5,3,8,10,8,3,2,7,9,4,8,9,5];
array [1..42] of int: X_INTRODUCED_125 = [8,5,7,8,7,4,10,3,9,2,8,7,1,9,2,9,10,8,8,4,8,3,4,2,4,6,7,7,3,9,6,10,10,9,4,9,10,2,4,7,2,6];
array [1..42] of int: X_INTRODUCED_126 = [9,3,4,10,4,3,7,8,7,9,5,7,9,8,9,1,3,2,6,5,10,1,4,6,8,1,1,7,10,4,10,1,3,8,10,3,2,10,1,9,4,8];
array [1..47] of int: X_INTRODUCED_128 = [7,8,5,4,3,10,9,1,1,4,7,5,9,3,6,2,6,8,1,7,7,5,7,1,1,6,8,1,10,4,1,7,4,2,5,3,5,2,9,9,4,8,10,7,10,5,5];
array [1..47] of int: X_INTRODUCED_129 = [1,9,3,4,3,10,5,5,4,7,5,9,5,7,9,9,4,8,9,1,3,2,6,5,4,2,6,9,8,1,1,4,10,1,3,8,10,8,3,3,2,10,1,9,1,8,5];
var 0..330: X_INTRODUCED_0;
var 0..330: X_INTRODUCED_1;
var 0..330: X_INTRODUCED_2;
var 0..330: X_INTRODUCED_3;
var 0..330: X_INTRODUCED_4;
var 0..330: X_INTRODUCED_5;
var 0..330: X_INTRODUCED_6;
var 0..330: X_INTRODUCED_7;
var 0..330: X_INTRODUCED_8;
var 0..330: X_INTRODUCED_9;
var 0..330: X_INTRODUCED_10;
var 0..330: X_INTRODUCED_11;
var 0..330: X_INTRODUCED_12;
var 0..330: X_INTRODUCED_13;
var 0..330: X_INTRODUCED_14;
var 0..330: X_INTRODUCED_15;
var 0..330: X_INTRODUCED_16;
var 0..330: X_INTRODUCED_17;
var 0..330: X_INTRODUCED_18;
var 0..330: X_INTRODUCED_19;
var 0..330: X_INTRODUCED_20;
var 0..330: X_INTRODUCED_21;
var 0..330: X_INTRODUCED_22;
var 0..330: X_INTRODUCED_23;
var 0..330: X_INTRODUCED_24;
var 0..330: X_INTRODUCED_25;
var 0..330: X_INTRODUCED_26;
var 0..330: X_INTRODUCED_27;
var 0..330: X_INTRODUCED_28;
var 0..330: X_INTRODUCED_29;
var 0..330: X_INTRODUCED_30;
var 0..330: X_INTRODUCED_31;
var 0..330: X_INTRODUCED_32;
var 0..330: X_INTRODUCED_33;
var 0..330: X_INTRODUCED_34;
var 0..330: X_INTRODUCED_35;
var 0..330: X_INTRODUCED_36;
var 0..330: X_INTRODUCED_37;
var 0..330: X_INTRODUCED_38;
var 0..330: X_INTRODUCED_39;
var 0..330: X_INTRODUCED_40;
var 0..330: X_INTRODUCED_41;
var 0..330: X_INTRODUCED_42;
var 0..330: X_INTRODUCED_43;
var 0..330: X_INTRODUCED_44;
var 0..330: X_INTRODUCED_45;
var 0..330: X_INTRODUCED_46;
var 0..330: X_INTRODUCED_47;
var 0..330: X_INTRODUCED_48;
var 0..330: X_INTRODUCED_49;
var 0..330: X_INTRODUCED_50;
var 0..330: X_INTRODUCED_51;
var 0..330: X_INTRODUCED_52;
var 0..330: X_INTRODUCED_53;
var 0..330: X_INTRODUCED_54;
var 0..330: X_INTRODUCED_55;
var 0..330: X_INTRODUCED_56;
var 0..330: X_INTRODUCED_57;
var 0..330: X_INTRODUCED_58;
var 0..330: X_INTRODUCED_59;
var 0..331: objective:: output_var;
array [1..60] of var int: s:: output_array([1..60]) = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
array [1..46] of var int: X_INTRODUCED_121 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_20,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_28,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_34,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_50,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
array [1..46] of var int: X_INTRODUCED_124 ::var_is_introduced  = [X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
array [1..42] of var int: X_INTRODUCED_127 ::var_is_introduced  = [X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_5,X_INTRODUCED_8,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_33,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_48,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_57];
array [1..47] of var int: X_INTRODUCED_130 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_54,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_59];
array [1..61] of var int: X_INTRODUCED_134 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59,objective];
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_4],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_5],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_10],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_3],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_17],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_50],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_27],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_56],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_58],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_6],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_20],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_32],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_7],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_11],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_36],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_8],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_9],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_27],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_6,X_INTRODUCED_12],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_6,X_INTRODUCED_14],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_6,X_INTRODUCED_24],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_31],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_42],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_43],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_8,X_INTRODUCED_40],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_8,X_INTRODUCED_59],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_9,X_INTRODUCED_15],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_9,X_INTRODUCED_16],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_9,X_INTRODUCED_19],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_13],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_28],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_30],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_11,X_INTRODUCED_23],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_11,X_INTRODUCED_25],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_11,X_INTRODUCED_54],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_12,X_INTRODUCED_44],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_12,X_INTRODUCED_52],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_12,X_INTRODUCED_55],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_13,X_INTRODUCED_15],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_13,X_INTRODUCED_49],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_14,X_INTRODUCED_58],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_15,X_INTRODUCED_18],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_16,X_INTRODUCED_22],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_16,X_INTRODUCED_37],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_17,X_INTRODUCED_48],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_18,X_INTRODUCED_21],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_18,X_INTRODUCED_25],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_18,X_INTRODUCED_26],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_20],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_34],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_20,X_INTRODUCED_30],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_20,X_INTRODUCED_39],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_20,X_INTRODUCED_43],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_21,X_INTRODUCED_53],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_22,X_INTRODUCED_29],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_23,X_INTRODUCED_45],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_24,X_INTRODUCED_26],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_24,X_INTRODUCED_28],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_24,X_INTRODUCED_34],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_25,X_INTRODUCED_51],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_26,X_INTRODUCED_59],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_27,X_INTRODUCED_40],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_28,X_INTRODUCED_31],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_28,X_INTRODUCED_38],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_29,X_INTRODUCED_47],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_30,X_INTRODUCED_33],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_30,X_INTRODUCED_42],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_31,X_INTRODUCED_45],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_32,X_INTRODUCED_35],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_32,X_INTRODUCED_43],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_32,X_INTRODUCED_44],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_33,X_INTRODUCED_35],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_33,X_INTRODUCED_36],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_33,X_INTRODUCED_53],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_34,X_INTRODUCED_37],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_34,X_INTRODUCED_41],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_34,X_INTRODUCED_42],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_35,X_INTRODUCED_38],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_36,X_INTRODUCED_41],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_36,X_INTRODUCED_44],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_36,X_INTRODUCED_52],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_37,X_INTRODUCED_45],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_38,X_INTRODUCED_40],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_38,X_INTRODUCED_41],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_38,X_INTRODUCED_46],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_39,X_INTRODUCED_49],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_40,X_INTRODUCED_55],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_41,X_INTRODUCED_59],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_42,X_INTRODUCED_48],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_43,X_INTRODUCED_46],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_43,X_INTRODUCED_49],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_44,X_INTRODUCED_47],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_44,X_INTRODUCED_58],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_45,X_INTRODUCED_48],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_46,X_INTRODUCED_54],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_47,X_INTRODUCED_56],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_48,X_INTRODUCED_52],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_49,X_INTRODUCED_51],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_50,X_INTRODUCED_51],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_50,X_INTRODUCED_53],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_51,X_INTRODUCED_55],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_52,X_INTRODUCED_57],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_53,X_INTRODUCED_54],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_54,X_INTRODUCED_56],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_55,X_INTRODUCED_57],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_56,X_INTRODUCED_57],-1);
constraint jacop_cumulative(X_INTRODUCED_121,X_INTRODUCED_120,X_INTRODUCED_119,36);
constraint jacop_cumulative(X_INTRODUCED_124,X_INTRODUCED_123,X_INTRODUCED_122,53);
constraint jacop_cumulative(X_INTRODUCED_127,X_INTRODUCED_126,X_INTRODUCED_125,44);
constraint jacop_cumulative(X_INTRODUCED_130,X_INTRODUCED_129,X_INTRODUCED_128,33);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_57,objective],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_58,objective],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_59,objective],-5);
solve :: int_search(X_INTRODUCED_134,smallest,indomain_min,complete) minimize objective;
