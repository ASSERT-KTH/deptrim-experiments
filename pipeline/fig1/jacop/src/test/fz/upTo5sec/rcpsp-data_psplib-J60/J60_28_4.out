objective = 84;
s = array1d(1..60, [0, 0, 0, 9, 1, 1, 13, 4, 11, 11, 1, 4, 18, 4, 18, 16, 16, 9, 25, 16, 25, 32, 21, 11, 18, 32, 32, 11, 19, 29, 34, 24, 13, 35, 25, 41, 41, 34, 49, 34, 56, 56, 35, 34, 42, 35, 56, 45, 43, 44, 9, 47, 46, 41, 66, 60, 75, 76, 45, 66]);
----------
==========
